src/services/has-many-getter.js

```diff
@@ -62,13 +62,13 @@ function HasManyGetter(model, association, opts, params) {
           return resolve(_.map(records, record => record[params.associationName]));
         });
     })
-      .then((recordIds) => {
+      .then(async (recordIds) => {
         const conditions = {
           $and: [{ _id: { $in: recordIds } }],
         };
 
         if (params.search) {
-          const conditionsSearch = searchBuilder.getConditions();
+          const conditionsSearch = await searchBuilder.getConditions();
           conditions.$and.push(conditionsSearch);
```

Comment:
has many getter don't work, isn't it?
https://www.loom.com/share/af5e47c964504785aa80793de0146132

Reply:
It works perfectly fine. DOn't forget that a relationship in mongo is not both way 😉 . So here it's simply because your relationship is empty in the user

Reply:
Ok I will have a look!